CONTRIBUTING.md

```diff
@@ -192,7 +192,7 @@ Each commit message consists of a **header**, a **body** and a **footer**.  The
 format that includes a **type**, a **scope** and a **subject**:
 
 ```
-<type>(<scope>): <subject>
+(revert: )<type>(<scope>): <subject>
```

Comment:
Hmm, the parentheses here on `(revert: )` are because it is optional, right? But the parentheses on `(<scope>)` are because they are needed in the string. Perhaps we should just not include the `revert` bit in this and just explain about reverts as you do very clearly below.